python/py_vapid/__init__.py

```diff
@@ -38,24 +43,27 @@ def __init__(self, private_key_file=None, private_key=None):
                     self._private_key = \
                         ecdsa.SigningKey.from_der(
                             base64.urlsafe_b64decode(private_key))
-            except Exception, exc:
+            except Exception as exc:
                 logging.error("Could not open private key file: %s", repr(exc))
                 raise VapidException(exc)
-            self._pubilcKey = self._private_key.get_verifying_key()
+            self._public_key = self._private_key.get_verifying_key()
```

Comment:
@jrconlin This change should also be pulled in since the old variable name was incorrect.